files/build_templates/buffers_config.j2

```diff
@@ -21,6 +21,10 @@ def
 {%-     set switch_role      = '' %}
 {%- endif -%}
 
+{%- if DEVICE_NEIGHBOR_METADATA is not defined %}
```

Comment:
Looks like a WA for some issue which affects all platforms. Consider having it as a separate PR?